lib/matplotlib/text.py

```diff
@@ -2246,16 +2242,13 @@ def draw(self, renderer):
         self._update_position_xytext(renderer, xy_pixel)
         self.update_bbox_position_size(renderer)
 
-        if self.arrow is not None:
-            if self.arrow.figure is None and self.figure is not None:
-                self.arrow.figure = self.figure
-            self.arrow.draw(renderer)
-
-        if self.arrow_patch is not None:
+        if self.arrow_patch is not None:   # FancyArrowPatch
             if self.arrow_patch.figure is None and self.figure is not None:
                 self.arrow_patch.figure = self.figure
             self.arrow_patch.draw(renderer)
 
+        # Draw text, including FancyBboxPatch, after FancyArrowPatch.
+        # Otherwise, a wedge arrowstyle can land partly on top of the Bbox.
         Text.draw(self, renderer)
```

Comment:
True, I did.  That is new code, and right next to another method in which super is used.  But the "Text.draw()" is not new, and it is one of several such lines in this module.  I'm inclined to think that "super-ifying" the module is out of scope for this PR.  I also note that not all parent class method calls in this module can be handled with super;  `Annotation.__init__` needs to call the `__init__` from each of its immediate parents.  The only way to avoid that would be to rewrite it so as to use composition rather than multiple inheritance.  In the long run that could bring a big gain in readability and maintainability, but given the present complexity of this entire family of code, it would be quite a job.